sql/core/src/test/scala/org/apache/spark/sql/internal/SQLConfSuite.scala

```diff
@@ -221,6 +221,19 @@ class SQLConfSuite extends QueryTest with SharedSQLContext {
       .sessionState.conf.warehousePath.stripSuffix("/"))
   }
 
+  test("DATA_SOURCE_TABLE_RELATION_CACHE_MAX_SIZE") {
+    assert(spark.conf.get(SQLConf.DATA_SOURCE_TABLE_RELATION_CACHE_MAX_SIZE) === 1000)
+    withSQLConf(SQLConf.DATA_SOURCE_TABLE_RELATION_CACHE_MAX_SIZE.key -> "2000") {
+      assert(spark.conf.get(SQLConf.DATA_SOURCE_TABLE_RELATION_CACHE_MAX_SIZE) === 2000)
+    }
```

Comment:
thanks! ah I just saw there had been [discussions about similar issues](https://github.com/apache/spark/pull/15539).

right now I don't see needs to change this case's `maxSize` value -- after all disabling the cache causes problems right now thus we cannot do it -- so let's put this config into `StaticSQLConf`? let me push a commit on this.